src/harness.js

```diff
@@ -306,7 +306,13 @@ class Harness {
     }
 
     this.log(`Filling ${answers.length} pages with answer: ${JSON.stringify(answers)}`);
-    const fillResult = await this.page.evaluate(async innerAnswer => await window.formFiller.fillAppForm(innerAnswer), answers);
+    const fillResult = await this.page.evaluate(async innerAnswer => {
+      try {
+        return await window.formFiller.fillAppForm(innerAnswer);
```

Comment:
sorry for the late feedback on this one: but i'm a little unsure on the value we are adding here...  why catch an exception to just rethrow it? what problem are we solving?

Reply:
If we don't catch the exception, we see a confusing error in the stack trace like

> Error: Evaluation failed: FormLogicError
> at ExecutionContext._evaluateInternal (node_modules/puppeteer-core/lib/cjs/puppeteer/common/ExecutionContext.js:221:19)
> at process._tickCallback (internal/process/next_tick.js:68:7)

When we catch and rethrow, the error in the stack trace makes more sense for the user like 

> Error: Evaluation failed: Error: Error encountered while filling form: FormLogicError: Could not evaluate: concat(../mother_name, ' baby'), message: Failed to execute 'evaluate' on 'Document': The string 'concat("Jane"s", " baby")' is not a valid XPath expression.

Reply:
can your test also make an assertion about not just the boilerplate you're adding but also the more descriptive error message like 'error.includes('mother_name')`?